app/views/searches/articles.html.erb

```diff
@@ -0,0 +1,27 @@
+<div data-page="<%= @articles.next_page %>" data-more="<%= @articles.show_more? %>">
+  <div id="articles">
+    <% @articles.files.each do |article| %>
+      <li class="card-container">
+        <a class="card card--has-img" target=_blank href=<%= sufia.download_path(article) %>>
+          <div class="card-figure-img">
+            <img src="<%= sufia.download_path(article, file: 'thumbnail') %>" alt="<%= article.title.first %>">
+          </div>
+          <div class="card-content">
+            <h3 class="card-content-title">
+              <%= article.title.first %>
+            </h3>
+            <h4 class="card-content-subtitle">
+              <%= article.production_names %>
+            </h4>
+          </div>
+        </a>
+      </li>
+    <% end %>
+  </div>
+
+  <div class="paging">
+    <% if @articles.show_more? %>
+    <a href="#" class="js-load-more-articles" data-page="<%= @articles.next_page %>">Load More</a>
+    <% end %>
+  </div>
+</div>
```

Comment:
How much duplication is there with the `_articles.html.erb` template?  Can we eliminate that somehow?  Also, the file naming might be somewhat confusing.  Similar comment for the other three resource types.